src/components/views/MigrationView.vue

```diff
@@ -356,6 +358,10 @@ export default {
                             console: change.name,
                         };
 
+                        if (change.message) {
+                            operation.details.push(change.message);
```

Comment:
`operation.details` isn't an array, is it?

Reply:
It is set to an empty array 4 lines above.